tests/trace/test_objects_query_builder.py

```diff
@@ -289,7 +302,7 @@ def test_make_objects_val_query_and_parameters():
     )
 
     expected_query = """
-        SELECT object_id, digest, any(val_dump)
+        SELECT object_id, digest, argMax(val_dump, created_at)
```

Comment:
why is this necessary?

Reply:
https://github.com/wandb/weave/pull/3866#discussion_r1985843854